src/codegen/codegen_acc_visitor.cpp

```diff
@@ -89,17 +89,19 @@ void CodegenAccVisitor::print_memory_allocation_routine() const {
     }
     printer->add_newline(2);
     auto args = "size_t num, size_t size, size_t alignment = 16";
-    printer->fmt_start_block("static inline void* mem_alloc({})", args);
-    printer->add_line("void* ptr;");
-    printer->add_line("cudaMallocManaged(&ptr, num*size);");
-    printer->add_line("cudaMemset(ptr, 0, num*size);");
-    printer->add_line("return ptr;");
-    printer->end_block(1);
+    printer->fmt_push_block("static inline void* mem_alloc({})", args);
```

Comment:
can avoid one fmt, by merging this line with the previous one?

Reply:
thanks. Taken into account in 6775a0a6781

Reply:
Do you still request changes in this contribution @alkino  ?